### DISCUSSION TITLE
Kismet and Grafana

### DISCUSSION THREAD

**Discussion by BOOMEL1** [WLAN-Pi/feedback#76] [General Inquiries]
Is there any more documentation on the use of Kismet and Grafana. I am even struggling setting up username and password (or establishing default). It would useful please to get some more screenshots / examples of their usage? 

BOOMEL

**Comment by JazzHex** [comment]
Refer to https://wlanpi.gitbook.io/cookbook.

You need to setup your own username and password for Kismet.

![image](https://github.com/user-attachments/assets/fd3aecdd-32bb-4cf9-8d80-2bf55e19d866)

Grafana details - >> https://wlanpi.gitbook.io/cookbook/4.-lab-wlan-pi-grafana/start-grafana

For more info refer to below youtube URLs

https://www.youtube.com/watch?v=uJm9T60wdVE&t=1s

https://www.youtube.com/watch?v=W1vBEy50B2M

**Comment by BOOMEL1** [comment]
Many thanks I will go through these. I have done an upgrade to latest software and since accessing web front end I get 

The address I am using is http://192.168.1.49 <http://192.168.1.49/> 

￼
-----------------------------
Regards,

Mike Lawson
07584202973

**Comment by JazzHex** [comment]
You should use **https**

**Comment by BOOMEL1** [comment]
Sorry still getting same error message with https://

-----------------------------
Regards,

Mike Lawson
07584202973

**Reply by JazzHex** [reply]
Is it displaying IP address on Front Panel Menu System?

![image](https://github.com/user-attachments/assets/c0a6a3bc-2fb5-4a28-aee7-dec5f0cb05b6)

**Comment by BOOMEL1** [comment]
YesSent from my iPhoneOn 5 Dec 2024, at 12:27, JazzHex ***@***.***> wrote:﻿
Is it displaying IP address on Front Panel Menu System?
image.png (view on web)

**Reply by JazzHex** [reply]
How is it going?

**Comment by BOOMEL1** [comment]
Sorry still not working. I used raspberry pi imager with new username and passoword. Going to try to re image without username and password. Regards,Mike LawsonOn 6 Dec 2024, at 22:38, JazzHex ***@***.***> wrote:﻿
How is it going?
